portfolio/src/main/webapp/script.js

```diff
@@ -67,5 +67,25 @@ function createListElement(name, text) {
  */
 function deleteCommentData() {
   const request = new Request('/delete-data', {method: POST});
-  fetch(request).then(result => getCommentData());
+  fetch(request).then(response => getCommentData());
+}
+
+/**
+ * Fetch user login status from LoginServlet.
+ */
+function getLoginStatus() {
+  fetch('/login').then(response => response.json()).then((userStatus) => {
+    if (userStatus.isLoggedIn) {
+      document.getElementById('comments-form-div').style.display = 'show';
```

Comment:
i think buttons should be defaulted to hidden. image this scenario: 
- logged out user visits your site
- fetch('/login') makes a request to the server (this method can take a while depending on users internet connection)
- since request still hasn't completed yet, a logged out user can see the post and delete buttons even though it should be hidden

if we default it to hidden, a signed in user might not see the buttons right away, which isn't a huge deal

Reply:
Ah that makes sense. I wrapped the delete button and the entire comments form in divs and set the default to hidden - then in the JS show them if logged in and hide them if logged out. When users are logged out, they'll still be able to load comments. Does that address this case?

Reply:
yea that makes sense!